scripts/environment.sh

```diff
@@ -57,6 +57,14 @@ export CAPM3_VERSION=${CAPM3_VERSION:-"v1beta1"}
 export M3PATH=${M3PATH:-"${HOME}/go/src/github.com/metal3-io"}
 export CAPM3_LOCAL_IMAGE="${CAPM3PATH}"
 
+export CAPIRELEASE="v1.1.3"
```

Comment:
in the feature test case the script release will export the latest release based on the version in this case it is v1beta1 so you do not need to hard code that I think you can remove it safely from here.
for the other `v0.4.8` yeah it is not exported but still better to not hard code it but use something like in the release script to get the latest based on the version v1alpha5.

Remember! The goal of this work is to not hard code releaseversions